sale_temporal_from_attributes/views/product_template.xml

```diff
@@ -0,0 +1,48 @@
+<odoo>
+
+    <record
+        id="action_compute_time_based_pricing_product_attributes"
+        model="ir.actions.server"
+    >
+            <field
+            name="name"
+        >Compute time based pricing from Product Attributes</field>
+            <field name="model_id" ref="product.model_product_template" />
+            <field name="binding_model_id" ref="product.model_product_template" />
+            <field name="state">code</field>
+            <field name="code">
+self = records or record  # In a Server Action
```

Comment:
@Nikul-OSI I would like this to be an actual code method, instead of a Server Action.
Maybe making `product_pricing_ids` a computable writable field is the easiest way.
Thanks.

Reply:
@dreispt removed server action and created method. 
**product_pricing_ids** This field already in template, so here I need to making computable?

Reply:
Yes, make it computable and add readonly=False.

Reply:
@dreispt Code updated!